client/pages/monthly-expense-list.jsx

```diff
@@ -25,6 +29,15 @@ class MonthlyExpenseList extends React.Component {
       .catch(err => console.error(err));
   }
 
+  handleDownload() {
+    const canvasImage = this.graph.current.toDataURL();
+    const doc = new JSPDF('landscape');
+    doc.addImage(canvasImage, 'JPEG', 10, 10, 280, 180);
```

Comment:
You should measure the actual canvas element. Your graph is super stretched out looking.